### PR TITLE
Cleaner branch

### PR DESCRIPTION
**Pull request by micha7a**
Clean most of theory related stuff. 

Stuff left to do:
 - remove unused folders 
 - add figures to GenerateAllFigures

### CODE REVIEW

**Comment by duembgen** [comment]
I added the figures used to create Figure 3. The three eps figures are obtained by saving each layer individually as an eps. Do we also want the final pdf of the figure here?

**Comment by micha7a** [comment]
> Do we also want the final pdf of the figure here?
I added .png files because it's tricky to display .pdf or .eps in the notebook/markdown/html. 

I also removed some of the results, but I think there is more to remove.

**Comment by micha7a** [comment]
Aaagh, more stuff, sorry. So, I forgot to add one script (`generate_noisy_results.py`) and I also noticed that `generate_results.py` is a module, not a script. Should we move it?

File: GUIDELINES.md [hunk 1]
```diff
@@ -76,4 +77,19 @@ add those as github submodules, for example:
...
+4. modules from within the project (e.g. trajectory)
+
+## Other 
+
+ - Do not use python shebang and do not put `__main__` in modules &mdash; create a new script if you need. 
+ - Save everything in figures/ or results/ (depending on if they are plots or data, respectively)
+ - Use `NotImplementError` either for functions that are under development (as in "not yet implemented") or for
+  abstract classes. For an option/flag that is not supported use `ValueError`.
+
+
```
Comment (duembgen) [inline GUIDELINES.md]: Nice!

**Pull request merged**